pandas/tests/extension/test_arrow.py

```diff
@@ -197,10 +206,71 @@ def test_loc_iloc_frame_single_dtype(self, request, using_array_manager, data):
         super().test_loc_iloc_frame_single_dtype(data)
 
 
+class TestBaseDtype(base.BaseDtypeTests):
+    def test_construct_from_string_own_name(self, dtype, request):
+        pa_dtype = dtype.pyarrow_dtype
+        if pa.types.is_timestamp(pa_dtype) and pa_dtype.tz is not None:
+            request.node.add_marker(
+                pytest.mark.xfail(
+                    raises=NotImplementedError,
+                    reason=f"pyarrow.type_for_alias cannot infer {pa_dtype}",
```

Comment:
do we have an issue for followup on these?

Reply:
No, but I can post an issue on the Arrow JIRA board.

If `pyarrow.type_for_alias` decides to implement better parsing, we should get this for free in a future version